spec/models/aliquot_spec.rb

```diff
@@ -149,22 +149,17 @@
     end
   end
 
-  describe '#sample_sheet_behaviour' do
-    it 'returns the hidden sample sheet behaviour when the aliquot has a hidden tag' do
-      aliquot = create(:aliquot, tag: create(:hidden_tag))
-      expect(aliquot.sample_sheet_behaviour.class).to eq(SampleSheetBehaviour::Hidden)
-    end
-
-    it 'returns the default sample sheet behaviour when the aliquot has a standard tag' do
-      aliquot = create(:aliquot, tag: create(:tag))
-      expect(aliquot.sample_sheet_behaviour.class).to eq(SampleSheetBehaviour::Default)
-    end
-
-    it 'returns the untagged sample sheet behaviour when the aliquot does not have a tag' do
-      aliquot = create(:aliquot, tag: nil)
-      expect(aliquot.sample_sheet_behaviour.class).to eq(SampleSheetBehaviour::Untagged)
-    end
-  end
+  # describe '#sample_sheet_behaviour' do
```

Comment:
Can this be uncommented? Looks like aliquot still has this method and we want to check no breaking changes.

Reply:
The sample sheet behaviour is already tested in the tagged? describe section.

This section tested implementation rather than behaviour which is why I removed it.